test_scripts/SDL4.6/DisplayName/001_SetDisplayLayout_DisplayNameParameter.lua

```diff
@@ -0,0 +1,191 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Seems better to just expand the [smoke test](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/Smoke/API/020_SetDisplayLayout_PositiveCase_SUCCESS.lua) in this case.